src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/TransformationInputToCaseTransformerTest.java

```diff
@@ -124,7 +126,7 @@ public void should_convert_to_case_data_with_new_fields_when_auto_case_creation_
     }
 
     @Test
-    public void should_convert_to_case_data_for_the_auto_case_creation_request() {
+    public void should_throw_for_the_auto_case_creation_request_and_warnings() {
```

Comment:
shouldn't be there two tests one for success and one when for unprocessable entity

Reply:
There is an existing test case `should_convert_to_case_data_with_new_fields_when_auto_case_creation_request_is_false`, I have just add `_and_warnings` to the method name.

Reply:
`should_throw_422_for_auto_case_creation_request_when_email_is_invalid` or something similar ?

Reply:
Renamed to `should_throw_422_for_auto_case_creation_request_when_validation_warnings` because `caseValidator` is a mock and not aware of emails or any other fields